backend/apps/mentorship/api/internal/nodes/mentee.py

```
@@ -27,4 +27,4 @@ def resolve_avatar_url(self) -> str:
     @strawberry.field(name="experienceLevel")
     def resolve_experience_level(self) -> str:
         """Get the experience level of the mentee."""
-        return self.experience_level if self.experience_level else "beginner"
+        return self.experience_level if self.experience_level else "beginner"
```

Ruff (0.14.10)

No newline at end of file

Add trailing newline

(W292)